mobile/src/.vscode/settings.json

```diff
@@ -8,5 +8,8 @@
   "editor.codeActionsOnSave": {
       "source.fixAll.eslint": true
   },
-  "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+  "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+  "[javascriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  }
```

Comment:
I don't think this change is necessary. Did you have the ESLint vscode extension?

Reply:
Yea I do